sanic/blueprints.py

```diff
@@ -77,14 +79,9 @@ def __init__(
     ):
         super().__init__(name=name)
 
-        self._apps: Set[Sanic] = set()
         self.ctx = SimpleNamespace()
-        self.exceptions: List[RouteHandler] = []
         self.host = host
-        self.listeners: Dict[str, List[ListenerType]] = {}
-        self.middlewares: List[MiddlewareType] = []
-        self.routes: List[Route] = []
-        self.statics: List[RouteHandler] = []
+        self.reset()
```

Comment:
Why is `reset()` in the middle of setting host, strict_slashes, etc?

Reply:
Fixed